core/shortcut_items.ts

```diff
@@ -81,6 +81,9 @@ export function registerDelete() {
   ShortcutRegistry.registry.register(deleteShortcut);
 }
 
+let copyData: ICopyData | null = null;
```

Comment:
If you store the data here, would this mean you can't copy a block with the context menu and paste it using a keyboard shortcut? if so that would not be desirable and would be a good reason to leave the state in the clipboard module.

Reply:
You can't copy blocks via the context menu, only duplicate them! And `copy` is internal, so external devs *shouldn't* have added a `copy` option to the context menu either.

Reply:
This is surprising yet logical...